Setup.hs

```diff
@@ -9,6 +9,8 @@ import System.IO (stdout, hFlush)
 -- For now let the user (or CI system) know that it
 -- is not unusual for there to be no output for a
 -- long time.
+main = defaultMain
```

Comment:
`defaultMain` is not in scope here, so building with `Setup.hs` will always fail.

Reply:
Ran into this myself, see https://github.com/ghcjs/jsaddle-dom/pull/24